aopy/postproc/base.py

```diff
@@ -554,7 +554,7 @@ def get_target_locations(preproc_dir, subject, te_id, date, target_indices):
     for i in range(len(target_indices)):
         trial_idx = np.where(trials['index'] == target_indices[i])[0][0]
         locations.append(trials['target'][trial_idx][[0,2,1]])
-    return np.array(locations)
+    return np.round(np.array(locations),2)
```

Comment:
can you just change the function to `return data.bmi3d.get_target_locations(preproc_dir, subject, te_id, date, target_indices)` so there are no differences between the two functions?